pipelines/DarkEnergyCamera/ApVerify.yaml

```diff
@@ -15,6 +15,15 @@ tasks:
     class: lsst.ap.association.DiaPipelineTask
     config:
       doPackageAlerts: True
+  transformDiaSrcCat:
+    class: lsst.ap.association.TransformDiaSourceCatalogTask
+    config:
+      doIncludeReliability: True
+  rbClassify:
+    class: lsst.meas.transiNet.RBTransiNetTask
+    config:
+      modelPackageName: 'rbResnet50-DC2'
+      connections.coaddName: parameters.coaddName
```

Comment:
Yes, I think `rbClassify` is now a permanent part of the pipeline. 
Moving it up (or down?) in the hierarchy of the yaml files sounds like a good idea. However, I think since this current PR has to do only with specific instruments (DC2 is already implemented and merged), let's bring this point up in a separate ticket. I would also think documenting this hierarchy (if it is not documented yet), would be beneficial for future debugging -- I remember @ebellm had comments along these lines too.

Reply:
The directory structure is described in [RFC-927](https://jira.lsstcorp.org/browse/RFC-927) and [RFC-775](https://jira.lsstcorp.org/browse/RFC-775). I could swear there was an open issue to put that convention in the pipelines(?) docs, but I can't find it.

As for the (already merged) PR, I'd like to object, for the record, to perpetuating known problems on the grounds that they could be fixed later. We already have tons of technical debt tickets that will never get scheduled, and in the meantime the code they're supposed to fix is enshrined and cargo-culted -- for example, because "it was already implemented that way".

Reply:
I am all for applying important updates ASAP; I'm also proposing giving them the time they deserve for being discussed and planned for.
In this case, I see the "let's implement this in another yaml" a design update suggestion (as opposed to a "known problem") that needs discussion; as mentioned, with @ebellm we were discussing the necessity of so many levels in the hierarchy the other day. I don't think under an approved PR there is enough space for such discussion -- more so when this PR is just about the implementation of the other part of what is already implemented.

How about you open a "move the ap_verify's rbClassify task to xx" ticket right now, so we can talk discuss all these there? (admittedly, I'm not sure about the best place for design discussions)

Reply:
Done as [DM-40210](https://jira.lsstcorp.org/browse/DM-40210) (which addresses _all_ the organizational problems that have built up with the pipelines over the years, not just `rbClassify`).